.rubocop.yml

```diff
@@ -142,184 +108,125 @@ Naming/MethodParameterName:
 # Performance
 Performance/Casecmp:
   Enabled: false
-
 Performance/StringReplacement:
   Enabled: false
 
 # Rails
 Rails/DynamicFindBy:
   Enabled: false
-
 Rails/FilePath:
   Enabled: false
-
 Rails/HasManyOrHasOneDependent:
   Enabled: false
-
 Rails/InverseOf:
   Enabled: false
-
 Rails/Output:
   Exclude:
     - 'db/migrate/*'
     - 'lib/changelog/**/*'
     - 'lib/tasks/*'
-
+Rails/ReversibleMigration:
+  Enabled: false
 Rails/ScopeArgs:
   Enabled: false
-
 Rails/SkipsModelValidations:
   Enabled: false
-
 Rails/TimeZone:
   Exclude:
     - 'lib/changelog/**/*'
-
 Rails/UnknownEnv:
   Environments:
     - development
-    - test
     - preview
-    - testing
-    - staging
     - production
-
+    - staging
+    - test
+    - testing
```

Comment:
These are different in KIDS and MAIN 🤔

Reply:
True, we can have it explicit in the rubocop yml on MAIN

Reply:
preview and testing are not available in KIDS. But I guess we just have to allow all of them here.

Reply:
just for `testing` & `preview`

Reply:
Ah yes, there is a way to merge/extend inherited rules! Let me check.

Reply:
Maybe like this? https://github.com/sofatutor/sofatutor/pull/14148